src/components/Documents/UploadDocs.tsx

```diff
@@ -155,15 +154,15 @@ class UploadDocs extends React.Component<Props, State> {
         this.state.documentTypeList &&
         this.state.documentTypeList.length < this.state.pdfFiles.length
       ) {
-        this.props.alert.show('Please categorize each document');
-        error = true;
+        // this.props.alert.show('Please categorize each document');
+        // error = true;
```

Comment:
why are we commenting out the error here?

Reply:
Sorry, I think this was blocking us earlier, but the error was unrelated. I've changed it back now